webapp/src/components/BarcodeModal.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <form class="modal-enclosure" data-testid="barcode-form">
+    <Modal
+      :model-value="modelValue"
+      :disable-submit="true"
```

Comment:
Maybe we could add something inside the Modal component to not display "Submit" here.

Reply:
Yeah, this is what I was trying to do with the blank footer, but alas :(